README.md

```diff
@@ -0,0 +1,65 @@
+# Youtube-VideoPlayerPage
+The goal here isn’t to produce an exact replica of the YouTube video page, but to focus on making the media elements show up. That means two things – embedding the YouTube video player into the page so it actually plays and showing little thumbnail images along the right side.
+
+![Youtube Project made by Gabriel & Emilio](Images/ProjectSS.png?raw=true "Youtube Project made by Gabriel & Emilio")
+
+Additional description about the project and its features.
+
+Built With
+HTML5,
+CSS.
+
+Getting Started
+First, visit this website https://raw.githack.com/. There, you will see an empty input where you can paste the link of our Github repositorie's html file. 
+This is going to give you a link right below a text "Use this URL in production"; please copy that link and paste it on your browser. ¡You will see our webpage once the browser finishes loading.
+
+Prerequisites
+Navigation bar
+It has an embedded image that represents a logo.
+The logo is aligned to the leftmost part of the navbar.
+It has an input tag that represents a search bar.
+The search bar is aligned to the middle of the navbar.
+It has a button tag that represents the search button.
+The search button is at the end (rightmost part) of the search bar.
+It has at least three embedded images on the rightmost part of the navbar that represent icons and/or an avatar.
+Video
+It is displayed.
+It shows the video title.
+It shows the number of views.
+It has like and dislike icons.
+It has a share button.
+It shows the avatar of the user who uploaded the video.
+It has a subscribe button.
+Sidebar Recommended videos
+It shows the recommended video’s thumbnail preview (as image not as an actual video).
+It shows the title of the video.
+It shows the number of views.
```

Comment:
These are not prerequisites. This section refers to when someone wants to clone your project and they need to have something installed before to be able to run it. You could change the title to Project details.